test/gie/builtins.gie

```diff
@@ -2183,10 +2209,35 @@ expect  -0.001790497 0.000895247
 accept  -200 -100
 expect  -0.001790496 -0.000895247
 
+accept -100.0 22.0
```

Comment:
those points in the inverse direction are all close to lat=lon=0. I would just remove them completely, but as suggested previously, after each expect statement above add a "roundtrip 1" line, that will check that when using the inverse path the values of expect are back projected to the ones of accept

Reply:
I added in the roundtrip option for all zones in my latest commit, but I am seeing failures with infs in zones 11 and 12.  I am stumped.  Do you see anything obvious with how I implemented the tests?